lib/src/go_true_exception.dart

```diff
@@ -1,18 +1,18 @@
-class GotrueError {
+class GoTrueException {
```

Comment:
Should be an `Exception`:

```suggestion
class GoTrueException implements Exception {
```